docs/source/essentials/schema.md

```diff
@@ -17,55 +30,212 @@ type Book {
 }
 
 type Author {
+  name: String
   books: [Book]
 }
 ```
 
-Take not that the schema is based upon the real world relationships between objects rather than the underlying data store. This real world basis provides the most intuitive interaction for the front-end. In addition, it ensures that the server understands what data will be retrieved together and can stay performant.
+It's important to note that these declarations express the relationships and the shape of the data to return, not where the data comes from or how it might be stored - which will be covered outside the SDL.
 
-Whether designing a new application or creating a GraphQL layer over your existing back-end, this basis in logical connections is the most important consideration is to define your schema. It is tempting to define a schema based on the current layout of your database, micro-services, or REST endpoints. Your GraphQL schema is going to be used across your entire organization and should facilitate the organization of your back-end services as well as enable front-end developers to easily discover what data can and should be retrieved together.
-
-> Often times mature architectures have been battle tested and optimized for certain views. There is a temptation to translate this data directly into the SDL. This can be an effective migration path provided there is a way to add new GraphQL schema types for new use cases. Often times, since these are newer features with less critical performance guarantees, it is acceptable to retrieve a portion of fields for  a given field from the optimized backing
+By drawing these logical connections in the schema, we can allow the client (which is often a human developer, designing a front-end) to see what data is available and request it in a single optimized query.
 
 ## Queries
```

Comment:
Something at the end of this section describing how GraphQL compares to rest REST. Its single request without overfetching/underfetching nature might be something that server implementers could appreciate. Pointing out that REST api documentation isn't necessary because of the client's inherent knowledge of the data shape might also resonate

Reply:
Added a bit in a8ad82ac3ec5a378a0c8a4ebc462edc7bcfcfd10, if you don't mind taking a look!